### DISCUSSION TITLE
Can I mount an .hds-image on Windows or open it on Pi terminal?

### DISCUSSION THREAD

**Discussion by hilburger** [akuker/RASCSI#1063] [Q&A]
Hey guys, 

I wonder if it's possible to mount or at least open an hdd image file? 

This would be help me because I'd like to update and/or change files inside of the hdd without the need to attach my Kurzweil sampler to a computer via SCSI. 

Thank you and best regards, 
Tom

**Comment by hilburger** [comment]
Okay, I have found via the free tool OSFMount ([https://www.osforensics.com/tools/mount-disk-images.html](https://www.osforensics.com/tools/mount-disk-images.html)). 
It can mount the raw imahe .hds file in read-only and write mode. 
Then you can access and add, delete, copy, backup the files and folders as you wish. 

The only downside is that you have to download the image file first via SFTP or FTP and load them up afterwards e.g. overwriting the old file. 

I could test this working with my Kurzweil K2000R and a 96MB test image. 

Maybe anyone has got a more elegant way o do such tasks?

**Reply by rdmark** [reply]
This method is contingent on the ability of the host OS to understand the file system on the disk image that you mount. In your case I'm assuming your Kurzweil device uses a standard FAT file system which is why Windows can read it without trouble.

Another piece of software for mounting images in Windows that I use myself:
https://sourceforge.net/projects/imdisk-toolkit/

To avoid the roundtrip over the network you can also mount the image directly on the Pi's Linux system with the `mount` command. Just make sure the image isn't actively used by PiSCSI when doing this.

**Reply by hilburger** [reply]
> This method is contingent on the ability of the host OS to understand the file system on the disk image that you mount. In your case I'm assuming your Kurzweil device uses a standard FAT file system which is why Windows can read it without trouble.
> 
> Another piece of software for mounting images in Windows that I use myself: https://sourceforge.net/projects/imdisk-toolkit/
> 
> To avoid the roundtrip over the network you can also mount the image directly on the Pi's Linux system with the `mount` command. Just make sure the image isn't actively used by PiSCSI when doing this.

Thanks! I'll try that 👍
